problems/removeVowels.js

```diff
@@ -6,8 +6,15 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+const removeVowels = str => {
+    let output = "";
+    for(let i = 0; i < str.length; i+=1){
```

Comment:
Try using a for of loop when going through strings, and factor to use toLowerCase and includes.